CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.24.1 (2022-03-26)
```

Comment:
No need to update this file. We handle this on version release. The changes here should be reverted.